job-service-container/pom.xml

```diff
@@ -41,6 +41,7 @@
         <caf.codec.impl>codec-json</caf.codec.impl>
         <caf.worker-queue.impl>worker-queue-rabbit</caf.worker-queue.impl>
         <dummyUrl>http://self:1234/job-service/v1</dummyUrl>
+        <postgres.db.port>54328</postgres.db.port>
```

Comment:
Why would we have a hard-coded port?  Surely this will mean that multiple builds can't run simultaneously.  And what's this change got to do with job cancellation?

Reply:
I was using this change to connect to a fixed port in my local while development. This was pushed by mistake. Reverted.